django_lti_tool_provider/migrations/0002_reduce_custom_key_length.py

```diff
@@ -32,8 +25,23 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.RunPython(
-            code=check_field_max_length_lteq_190,
-            reverse_code=reverse_migration
+            code=check_field_max_length_lteq_190
+        ),
+        migrations.RunSQL(
+            sql=migrations.RunSQL.noop,
+            state_operations=[
+                # This is a shim. Django is oh-so-efficient, so if it detects
+                # that the database should be in a state based on previous migrations,
+                # it won't perform the operations to move to that state. This tells
+                # Django "hey, I'm in this different state" without actually modifying
+                # the database. This is not a reversible operation.
+                migrations.AlterField(
+                    model_name='ltiuserdata',
+                    name='custom_key',
+                    field=models.CharField(default=b'', max_length=400)
+                )
+            ],
+            elidable=False
```

Comment:
This causes an error on Django < 1.10 (dalite-ng uses 1.8):

```
  File "/edx/src/django-lti-tool-provider/django_lti_tool_provider/migrations/0002_reduce_custom_key_length.py", line 44, in Migration
    elidable=False
TypeError: __init__() got an unexpected keyword argument 'elidable'
```

After removing that line, it worked fine.